content/tutoren.pug

```diff
@@ -96,5 +97,21 @@ block main
 			// ['Wendy Yi', 'Bachelor Informatik', 2017],
 			// ['Yussuf Khalil', 'Master Informatik', 2018]
 		]
-	each Tutorarray in Tutorenarrays.sort((t1, t2) => t1[0].localeCompare(t2[0], 'de-DE'))
+	
+	- // Sortiere Tutoren nach der Vorhandenheit eines Bildes alphabetisch
+	-
+		const sortedTutorenarrays = Tutorenarrays.sort((t1, t2) => {
+			var fs = require('fs');
+			var t1ImageExists = fs.existsSync('images/tutoren/' + t1[0] + '.jpg');
+			var t2ImageExists = fs.existsSync('images/tutoren/' + t2[0] + '.jpg');
```

Comment:
The pendant in me needs to  comment that you should _really_ cache these filesystem accesses. But then again, it probably does not make much of a notable difference.

Reply:
Indeed, I thought of refactoring already, but came to the conclusion that it won't matter for those small numbers.